src/LightBox.js

```diff
@@ -50,11 +50,17 @@ export default class LightBox extends Component {
   }
 
   componentDidMount() {
-    addListener('message', this.onMessage)
+    if (typeof window !== 'undefined') {
+        const addListener = window.addEventListener || window.attachEvent
+        addListener('message', this.onMessage)
+    }
   }
 
   componentWillUnmount() {
-    removeListener('message', this.onMessage)
+    if (typeof window !== 'undefined') {
```

Comment:
The lifecycle method `componentWillUnmount` is not called when rendered on the server, you can safely remove this check.

Reply:
Yes that is correct
I will forward it...